vnext/JSI.Desktop.UnitTests/JsiRuntimeUnitTests.cpp

```diff
@@ -0,0 +1,1072 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+#include "JsiRuntimeUnitTests.h"
+
+#include <jsi/jsi.h>
+#include <utilities.h>
+
+#include <gtest/gtest.h>
+
+#include <stdlib.h>
+
+// Note: We cannot use unistd.h here because it is a Unix header.
+//       Instead we use the standard chrono header.
+//#include <unistd.h>
+#include <chrono>
+
+#include <thread>
+#include <functional>
+#include <unordered_map>
+#include <unordered_set>
+
+using namespace facebook::jsi;
+using namespace facebook::react::utilities;
+
+
+TEST_P(JsiRuntimeUnitTests, RuntimeTest) {
```

Comment:
>TEST_P [](start = 0, length = 6)

these tests should be in some sort of namespace

Reply:
We chatted offline about this.